File: Documents/NetBeansProjects/JavaApplication2/src/com/mycompany/projetoback/Cliente.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package com.mycompany.projetoback;

/**
 *
 * @author pedro
 */
public class Cliente {
    private String nome;
    private String senha;
    private int idade;
    private boolean tipo;
    private int id;
    
    public Cliente() {
        this.nome = "";
        this.idade = 0;
        this.tipo = false;
        this.senha = "";
    }
    
    public Cliente(String nome, String senha, int idade, boolean tipo) {
        this.nome = nome;
        this.idade = idade;
        this.tipo = tipo;
        this.senha = senha;
    }

    public Cliente(String nome, String senha) {
        this.nome = nome;
        this.senha = senha;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getNome() {
        return nome;
    }

    public void setNome(String nome) {
        this.nome = nome;
    }

    public int getIdade() {
        return idade;
    }

    public void setIdade(int idade) {
        this.idade = idade;
    }

    public boolean getTipo() {
        return tipo;
    }

    public void setTipo(boolean tipo) {
        this.tipo = tipo;
    }

    public String getSenha() {
        return senha;
    }

    public void setSenha(String senha) {
        this.senha = senha;
    }
    
}

File: Desktop/cinem/classes/Cliente.py
class Cliente:
    def __init__(self, Id, nome, emails, senhas, sobrenomes, idades, tipofuncionario):
        self.Id = Id
        self.nome = nome
        self.emails = emails
        self.senhas = senhas
        self.sobrenomes = sobrenomes
        self.idades = idades
        self.tipofuncionario = tipofuncionario

    def getId(self):
        return self.__Id
    def setId(self, Id):
        self.__Id = Id
    def getNome(self):
        return self.__nome
    def setNome(self, nome):
        self.__nome = nome
    def getEmails(self):
        return self.__emails
    def setEmails(self, emails):
        self.__emails = emails
    def getSenha(self):
        return self.__senhas
    def setSenha(self, senhas):
        self.__senhas = senhas
    def getSobrenome(self):
        return self.__sobrenomes
    def setSobrenomes(self, sobrenomes):
        self.__sobrenomes = sobrenomes
    def getIdade(self):
        return self.__Idade
    def setIdade(self, idade):
        self.__idade = idade
    def getTipofuncionario(self):
        return  
